src/runtime/driver/mod.rs

```diff
@@ -52,7 +52,7 @@ impl Driver {
 
     // only used in tests rn
     #[allow(unused)]
-    fn num_operations(&self) -> usize {
+    pub(super) fn num_operations(&self) -> usize {
```

Comment:
Would be nice if things like this were exposed to the tracing tool, tokio console.

Reply:
I think that this is a good idea to expose eventually.